openreferee_server/operations.py

```diff
@@ -87,3 +94,32 @@ def cleanup_event(event):
     session = setup_requests_session(event.token)
     cleanup_event_tags(session, event)
     cleanup_file_types(session, event)
+
+
+def process_editable_files(session, files, endpoints):
+    uploaded = defaultdict(list)
+    for _file in files:
+        if _file['filename'].split(".")[-1] == 'pdf':
```

Comment:
As much as I like single quotes (and I still want to fork `black` so we can use black with quote normalization to single quotes!), I think currently this project uses black with quote normalization enabled so I think you should run black on the code and let it do its dirty work of switching to double quotes.

Or we could just disable quote normalization and stick with the nicer single quotes :)

Reply:
Forgot to setup the pre-commit hook.